source/features/preserve-file-finder-term.tsx

```diff
@@ -0,0 +1,48 @@
+import features from '../libs/features';
+import select from 'select-dom';
+import elementReady from 'element-ready';
+
+const FILE_FINDER_INPUT_SELECTOR = '.js-tree-finder > .breadcrumb > #tree-finder-field';
+
+const unloadHandler = (): void => {
+	const inputElement = select<HTMLInputElement>(FILE_FINDER_INPUT_SELECTOR);
```

Comment:
The selector can be inlined here, it doesn’t need to be a constant. In the other feature as well.

Reply:
I still see `FILE_FINDER_INPUT_SELECTOR` in both files

Reply:
Ok, I'd thought you meant the constant `inputElement` was not required. My bad.